deepr/example/configs/config.json

```diff
@@ -189,6 +189,15 @@
                 "fetch": "y_pred",
                 "new_names": {"x": "inputs/x"}
             },
+            {
+                "type": "deepr.jobs.ExportXlaModelMetadata",
+                "path_optimized_model": "$paths:path_optimized_model",
+                "path_metadata": "$paths:path_optimized_model",
+                "graph_name": "_model.pb",
+                "metadata_name": "_meta.pbtxt",
+                "feed_shapes": {"inputs/x": [1]},
+                "fetch_shapes": {"y_pred": null}
```

Comment:
is null truly valid for XLA? why not `[]`

Reply:
yeah, it's None, to not have shape for the output